src/Game.java

```diff
@@ -0,0 +1,115 @@
+import java.util.*;
+
+public class Game {
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+    private static final int MAX_ERRORS = 8;
+
+    private Scanner scanner = new Scanner(System.in);
+
+    private String wordToFind;
+    private char[] guessedWord;
+    private int mistakesMade;
+    private ArrayList<String> words = new ArrayList<>();
+
+    Keywords keywords = new Keywords();
+    WordsFromFile wordsFromFile = new WordsFromFile();
+
+    public void hangman() {
+        int option = -1;
+        while (option != EXIT) {
+
+            printOption();
+
+            option = scanner.nextInt();
+            scanner.nextLine();
+
+            switch (option) {
+                case EXIT:
+                    System.out.println("Bye bye!");
+                    scanner.close();
+                    break;
+                case PLAY:
+                    play();
+                    break;
+                default:
+                    System.out.println("Wybrałeś nieprawidłową opcję");
+            }
+        }
+    }
+
+    private void printOption(){
+        System.out.println("Wybierz opcję:");
+        System.out.println("1 - graj");
+        System.out.println("2 - koniec programu");
+    }
+
+    private void newGame() {
+        mistakesMade = 0;
+        words.clear();
+        wordToFind = wordsFromFile.nextWordToFind();
+        guessedWord = new char[wordToFind.length()];
+
+        Arrays.fill(guessedWord, '*');
```

Comment:
Ah, ok, I think i understand now. guessedWord is an array bc it's easier to replace * with actual characters.

Reply:
Si, claro